website/source/docs/provisioning/ansible_intro.html.md

```diff
@@ -239,6 +239,7 @@ config.vm.network :private_network, ip: "192.168.111.222"
 **Notes:**
 
   - The machine names in `Vagrantfile` and `ansible.inventory_path` files should correspond, unless you use `ansible.limit` option to reference the correct machines.
+  - The `ansible.inventory_path` option doesn't override the `ansible.implicit` command. To allow access to all available machines in the inventory, it is necessary to set `ansible.limit = "all"`.
```

Comment:
> The `ansible.inventory_path` option doesn't override the `ansible.implicit` command.

I don't think this part makes sense. There's no `ansible.implicit` command. Instead, I believe this is more about the _implicit_ notion that it works on a single guest by default, rather than all guests. Instead, this should probably read something like:


 > - The `ansible.inventory_path` option by default is only scoped to apply to a single guest in the inventory file, and does not apply to all defined guests. To allow access to all available machines in the inventory, it is necessary to set `ansible.limit = "all"`.

Reply:
Ok, it makes sense.
I pushed a change.